src/main/scala/uk/gov/hmrc/ct/accounts/frs10x/boxes/AC8081.scala

```diff
@@ -16,17 +16,17 @@
 
 package uk.gov.hmrc.ct.accounts.frs10x.boxes
 
-import uk.gov.hmrc.ct.accounts.frs10x.retriever.Frs10xAccountsBoxRetriever
+import uk.gov.hmrc.ct.accounts.frs10x.retriever.{Frs10xAccountsBoxRetriever, Frs10xDormancyBoxRetriever}
 import uk.gov.hmrc.ct.box._
 
 case class AC8081(value: Option[Boolean]) extends CtBoxIdentifier(name = "For the year ending <<POA END DATE>> the company was entitled to exemption under section 477 of the Companies Act 2006 relating to small companies.")
   with CtOptionalBoolean
   with Input
-  with ValidatableBox[Frs10xAccountsBoxRetriever]
+  with ValidatableBox[Frs10xAccountsBoxRetriever with Frs10xDormancyBoxRetriever]
   with Validators {
 
-  override def validate(boxRetriever: Frs10xAccountsBoxRetriever): Set[CtValidation] = {
-    collectErrors(
+  override def validate(boxRetriever: Frs10xAccountsBoxRetriever with Frs10xDormancyBoxRetriever): Set[CtValidation] = {
+    failIf(!boxRetriever.acq8999().orFalse) (
       validateAsMandatory(this)
```

Comment:
8081 is an existing box and I didnt change the validation - only the the conditions under which it occurs.
I can make it check for ==true but Id need to change all the existing declaration boxes to do the same to make it consistent